pydy/system.py

```diff
@@ -610,3 +610,42 @@ def _Kane_constant_symbols(self):
                 *self._Kane_inlist_insyms()))
         constants.remove(dynamicsymbols._t)
         return constants
+
+    def add_gravity(self, direction=None):
```

Comment:
@tarzzz That's a good point. The ability to add forces and torques should be defined first. If we think that gravity warrants a special case and deserves it's own method, then we can add that. Gravity is a case that is use ubiquitously, i.e apply the same force to all bodies/particles in the system.